charts/growthbook/values.yaml

```diff
@@ -0,0 +1,167 @@
+  name: ""
+
+growthbook:
+  nodeEnv: production
+  externalMongodbUri:
+  appOrigin: https://my-app-origin.io:443
+  apiHost: https://my-api-host.io:443
+  email:
+    enabled: false
+    host: host.example.com
+    port: 25
+    hostUser: user
+    hostPassword: password
+    sender: sender@example.com
+  persistence:
+    enabled: false
+    ## what type volume, possible options are [existing, emptyDir, dynamic] dynamic for Dynamic Volume Provisioning, existing for using an existing
+    type: emptyDir
+    ## Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    storageClassName: "-"
+    ## Persistent Volume Access Mode
+    ##
+    accessModes:
+      - ReadWriteMany
+    ## Persistent Volume size
+    ##
+    storage: 3Gi
+    ## Use an existing PVC
+    ##
+    # existingClaim:
+
+tolerations: []
+
+affinity: {}
+
+# Set your nodeSelector setting
+nodeSelector: {}
+
+volume:
+  name: uploads-persistent-storage
+  mountPath: /usr/local/src/app/packages/back-end/uploads
+
+service:
+  type: ClusterIP
+
+ingress:
+  enabled: false
+  name: growthbook-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/enable-cors: "true"
+    # Specify your app-origin with 'https://' prefix
+    nginx.ingress.kubernetes.io/cors-allow-origin: "https://my-app-origin.io"
+    nginx.ingress.kubernetes.io/cors-allow-headers: "Authorization,Referer,sec-ch-ua,sec-ch-ua-mobile,sec-ch-ua-platform,User-Agent,X-Organization,Content-Type"
+  # Specify your app-origin, api-host without 'https://' prefix
+  appOriginName: my-app-origin.io
+  apiHostName: my-api-host.io
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+
+# Disable if you already set up mongodb, you should also disable dependencies in Chart.yaml.
+# Make sure you specify right mongodb credential in secret
+# Example below assumes prometheus is running at sys-monitoring namespace
+# For more configurations and explanations, visit https://github.com/bitnami/charts/tree/master/bitnami/mongodb
+mongodb:
+  # -- Install MongoDB?
+  enabled: true
+  # For now we only support a db installed with a subchart
+
+  auth:
+    rootUser: root
+    rootPassword: password
+    username: growthbook
+    password: growthbook
+    database: growthbook-db
+  initdbScripts:
+    grant-role.js: |
+      use growthbook-db
+      db.grantRolesToUser("{{ .Values.auth.username }}", [{ role: 'readWrite', db: "{{ .Values.auth.database }}" }])
+
+gatekeeper:
```

Comment:
Do we need Gatekeeper here? Does it not have built-in OIDC integration?

Reply:
Growthbook only provides SSO for Enterprise Cloud Edition..Having Gatekeeper will protect public access to our prod (like n8n deployment I guess..)